src/ciadmin/generate/worker_pools.py

```diff
@@ -195,10 +195,25 @@ def get_azure_provider_config(
         {"implementation": implementation},
     )
     worker_config = merge(worker_config, config.get("worker-config", {}))
+    gw_config = worker_config["genericWorker"]["config"]
     if azure_config.get("wst_server_url"):
-        worker_config["genericWorker"]["config"].setdefault(
-            "wstServerURL", azure_config["wst_server_url"]
-        )
+        gw_config.setdefault("wstServerURL", azure_config["wst_server_url"])
+
+    # Populate some generic-worker metadata.
+    metadata = {}
+    has_sbom = image.get(provider_id, "sbom") in (None, True)  # defaults None to True
+    if has_sbom and "sbom_url_tmpl" in azure_config:
+        context = image.clouds[provider_id].copy()
+
+        # TODO Ensure the sbom urls match defined names, then remove this.
```

Comment:
Is this a TODO that you need to address before landing?

Reply:
No, the names as defined in `worker-images.yml` use underscore, while the urls use dashes. So the comment was more about making those two things match each other. I'll make it a bit more clear.